base/ocsp/src/main/java/com/netscape/cms/ocsp/DefStore.java

```diff
@@ -873,12 +818,12 @@ public X509CRLImpl getX509CRLImpl() {
 
 class DefStoreCRLUpdater extends Thread {
 
-    public static org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DefStoreCRLUpdater.class);
+    public static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DefStoreCRLUpdater.class);
 
-    private Hashtable<String, CRLIPContainer> mCache = null;
+    private Map<String, CRLIPContainer> mCache = null;
```

Comment:
This one is pointing to `mCacheCRLIssuingPoints`, so it should be the same type.